data/opentelemetry/nodejs-clone-application.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Understanding and Cloning the Sample Application - OpenTelemetry NodeJS
+slug: nodejs-clone-application
+date: 2024-06-05
+tags: [opentelemetry-tutorials]
+authors: [yuvraj]
+description: Learn how to clone and set up a NodeJS microservices application for implementing OpenTelemetry and SigNoz. This guide covers repository cloning, Docker setup, adding mock data, and testing the application, providing a foundation for enhanced observability in microservices...
+# image: /img/blog/2023/07/opentelemetry_golang_cover-min.jpg
+keywords: [opentelemetry, opentelemetry nodejs, distributed tracing, observability, nodejs application monitoring, nodejs instrumentation, signoz]
+---
+
```

Comment:
Should we add link to previous article at the start and next article at the end? @ankit01-oss  @zriyansh 

Because currently we don't have a UI it felt disconnected to me when reading the blogs on preview.

Reply:
<img width="866" alt="image" src="https://github.com/SigNoz/signoz-web/assets/18325767/3eb8c925-1318-4c08-ac75-270e8d45f46b">

Ideally this ToC should have the entire series structure

Reply:
yes @yuvraajsj18 . For now we should add a block at the top and bottom with all articles in the series listing down all articles in the series and showing users where they are currently. 

You can take this up post all articles are live and can do it one PR.

Reply:
Understood, Thanks.